sdk/file_sdk/XMPFileFormat/xmp_file_format.cpp

```diff
@@ -8,7 +8,15 @@ XMPFileFormat::XMPFileFormat(shared_ptr<IStream> file, const string& extension)
 {
 }
 
-const vector<Tag>& XMPFileFormat::ReadTags() {
+const vector<Tag> XMPFileFormat::ReadTags() {
+  return mTags;
+}
+
+string XMPFileFormat::GetOriginalExtension() {
+  throw std::exception("not implemnted");
+}
+
+void XMPFileFormat::Commit(shared_ptr<IStream> file, string& newExtension) {
   throw std::exception("not implemnted");
```

Comment:
typo :)

Reply:
Not my typo, i copied it...
will fix in all places

---
In reply to: [135754201](https://github.com/AzureAD/rms-sdk-for-cpp/pull/136#discussion_r135754201) [](ancestors = 135754201)